Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp

```diff
@@ -190,53 +190,54 @@ void HacksWidget::SaveSettings()
-  static const char TR_SKIP_EFB_CPU_ACCESS_DESCRIPTION[] =
-      QT_TR_NOOP("Ignore any requests from the CPU to read from or write to the EFB.\nImproves "
-                 "performance in some games, but might disable some gameplay-related features or "
-                 "graphical effects.\n\nIf unsure, leave this unchecked.");
+  static const char TR_SKIP_EFB_CPU_ACCESS_DESCRIPTION[] = QT_TR_NOOP(
+      "Ignores any requests from the CPU to read from or write to the EFB.\n\n"
+      "Improves performance in some games, but might disable some gameplay-related "
+      "features or graphical effects.\n\nIf unsure, leave unchecked.");
   static const char TR_IGNORE_FORMAT_CHANGE_DESCRIPTION[] = QT_TR_NOOP(
-      "Ignore any changes to the EFB format.\nImproves performance in many games without "
-      "any negative effect. Causes graphical defects in a small number of other "
-      "games.\n\nIf unsure, leave this checked.");
+      "Ignores any changes to the EFB format.\n\nImproves performance in many games "
+      "without any negative effect. Causes graphical defects in a small "
+      "number of other games.\n\nIf unsure, leave checked.");
   static const char TR_STORE_EFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to "
-      "RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores EFB Copies exclusively on the GPU, bypassing system memory.\nEnabled: EFB "
+      "Copies to Texture\nDisabled: EFB Copies to RAM (and Texture)\n\nCauses "
+      "graphical defects in a small number of games that need to readback from "
+      "memory.\n\nIf unsure, leave checked.");
   static const char TR_ACCUARCY_DESCRIPTION[] = QT_TR_NOOP(
-      "The \"Safe\" setting eliminates the likelihood of the GPU missing texture updates "
-      "from RAM.\nLower accuracies cause in-game text to appear garbled in certain "
-      "games.\n\nIf unsure, use the rightmost value.");
+      "Adjusts the accuracy at which the GPU receives texture updates from RAM. "
+      "\n\nThe \"Safe\" setting eliminates the likelihood of the GPU missing texture updates "
+      "from RAM.\n\nLower accuracies cause in-game text to appear garbled in certain "
+      "games.\n\nIf unsure, select the rightmost value.");
 
   static const char TR_STORE_XFB_TO_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Stores XFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
-      "in a small number of games that need to readback from memory.\n\nEnabled = XFB Copies to "
-      "Texture\nDisabled = XFB Copies to RAM "
-      "(and Texture)\n\nIf unsure, leave this checked.");
+      "Stores XFB Copies exclusively on the GPU, bypassing system memory. "
+      "\nEnabled: XFB Copies to Texture\nDisabled: XFB Copies to RAM (and Texture)\n\nCauses "
```

Comment:
This restructuring is unwise. The Enabled:/Disabled: subsections should be kept entirely separate from the description itself, as they are callbacks to older settings to assist users in the transition and the intention is to remove them at a later date some point after 6.0.